docs/source/advanced-guide/upgrade-cost-analysis-generic-models.rst

```diff
@@ -0,0 +1,13 @@
+**********************************
```

Comment:
@jgu2 Note that I added this. The URBANopt team wants to have access to our JSON schemas. Let me know if you'd prefer this in a different place.

Reply:
It's good to me.